src/components/ebay-notice/index.js

```diff
@@ -16,30 +18,53 @@ const constants = {
 
 const defaults = {
     type: 'page',
-    status: 'confirmation'
+    status: 'priority',
+    hidden: false,
+    dismissible: false
 };
 
-function getTemplateData(state, input) {
+function getInitialState(input) {
+    const hidden = input.hidden || defaults.hidden;
```

Comment:
In the docs, it says this is not supported as an initial attribute, in which case it should not use `input.hidden`.

Reply:
How is it that one is supposed to change the hidden state from outside this component? should we just get the component manually in code and change it?

If this is the case. We should consider that a more declarative way would be to hook to `onInput` so that whenever the parent re-renders this component with different attributes, it changes.

Reply:
That's a good point, that `input` can be for both initial attributes and stateful properties. We'll need to figure out a general solution for cases like this.